frontend/src/pages/HomePage/components/OverviewItem/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import * as S from './styles';
+
+export interface OverviewItemStyleProps {
+  direction: React.CSSProperties['flexDirection'];
+}
+
+interface OverviewItemProps extends OverviewItemStyleProps {
+  imageSrc: string;
+  title: string;
+  description: string[];
+}
+
+const OverviewItem: React.FC<OverviewItemProps> = ({ direction, imageSrc, title, description }) => {
```

Comment:
```suggestion
const OverviewItem: = ({ direction, imageSrc, title, description }:OverviewItemProps) => {
```
이라고 하지 않고  `React.FC<OverviewItemProps> `을 사용한 이유가 있나요?